python/pyproject.toml

```
@@ -13,16 +13,52 @@
 # limitations under the License.
 
 [build-system]
-
+build-backend = "setuptools.build_meta"
 requires = [
     "wheel",
-    "setuptools",
+    "setuptools>=61.0.0",
     "cython>=0.29,<0.30",
     "scikit-build>=0.13.1",
     "cmake>=3.23.1,!=3.25.0",
     "ninja",
-    "cuda-python>=11.7.1,<12.0"
+    "cuda-python>=11.7.1,<12.0",
+    "tomli; python_version < '3.11'",
+]
+
+[project]
+name = "rmm"
+version = "23.04.00"
+description = "rmm - RAPIDS Memory Manager"
+readme = { file = "README.md", content-type = "text/markdown" }
+authors = [
+    { name = "NVIDIA Corporation" },
+]
+license = { file = "LICENSE" }
```

Would stick with the metadata here as we can add the license file in a different field

```suggestion
license = { text = "Apache 2.0" }
```

---

I'm going to close these two new threads so that we can consolidate the discussion on the older one.